src/routes/welcome/components/Layout.jsx

```diff
@@ -64,14 +65,19 @@ export const LoadSafe = ({ size, provider }: SafeProps) => (
   </Button>
 )
 
-const Welcome = ({ provider }: Props) => (
-  <Block className={styles.safe}>
-    <Heading tag="h1" weight="bold" align="center" margin="lg">
-      Welcome to
+const Welcome = ({ provider, isOpenSafe }: Props) => {
+  const headingText = isOpenSafe ? (
```

Comment:
Wdyt about using `isOldMultisigMigration` as var name? I personally don't find `isOpenSafe` really meaningful.

Reply:
Yes, that's makes more sense to me also, naming is hard. Will change it :)